srcpkgs/povray/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'povray'
+pkgname=povray
+version=3.7.0.7
+revision=1
+_v=${version%.*.*}
+hostmakedepends="boost-devel git glu-devel autoconf automake zlib-devel libpng-devel libjpeg-turbo-devel tiff-devel SDL-devel"
+depends="tiff libpng boost openexr SDL2"
+short_desc="Script-based rayracer for 3D graphic production"
+maintainer="Brenton Horne <brentonhorne77@gmail.com>"
+license="AGPL-3.0-or-later"
+homepage="http://povray.org/"
+distfiles="https://github.com/POV-Ray/povray/archive/v${version}.tar.gz"
+checksum=085746e891edbb2cfda22bb2a8b86043bd680a68ad9121bc568118c730ace7b9
+
+do_configure() {
+	cd $wrksrc/unix
+	./prebuild.sh
+	cd ..
+	./bootstrap
+	./configure \
+	LIBS="-lboost_system -lboost_thread" \
+	COMPILED_BY="Void" \
+	--sysconfdir=/etc \
+	--prefix=/usr
+}
+
+do_build() {
+	make CXXFLAGS+="-std=c++11 -lboost_system -lboost_thread -w"
+}
+
+do_install() {
+	vmkdir usr/share/${pkgname}-${_v}
+	vmkdir usr/share/doc/${pkgname}-${_v}
+	cp -r icons include ini scenes scripts "${DESTDIR}/usr/share/$pkgname-$_v"
+	cp -r doc/html/* ${DESTDIR}/usr/share/doc/${pkgname}-${_v}
+	vbin unix/povray usr/bin/povray
+	rm -rf "${DESTDIR}/etc/povray/$_v/povray.conf"
+	vinstall povray.conf 644 "etc/povray/$_v/povray.conf"
```

Comment:
```povray: error while reading/opening config file /etc/povray/3.7/povray.conf: Is a directory```
`vinstall  povray.conf 644 "etc/povray$_v"` should be correct. Has to be fixed also in the two other vinstall lines.